about.html

```diff
@@ -0,0 +1,166 @@
+          Pharmcists in the Pharmacy space, the ACPN conference provides a
+          focused and unique opportunity to learn, share best practices, network
+          with peers, and interact with other healthcare providers in the
+          pharmaceutical space. At the ACPN Conference, community Pharmacists
+          can learn how best to offer pharmaceutical care, gain insights on more
+          efficient management processes and become aware of new technology and
+          techniques, which will ultimately help them and their companies: Sell
+          more products, Reduce costs, and Increase efficiencies
+        </p>
+        <p class="about-venue">
+          Please contact us via email for any further question and
+          clarifications about the ACPN conference 2023
+        </p>
+        <a class="email" href="#">info@acpn2023.org</a>
+      </section>
+    </main>
+    <!-- MAIN ENDS -->
+
+    <!-- SUB-SECTION STARTS -->
+    <section class="ending">
+      <div class="holder">
+        <h2>ACPN Conference 2023 Logo</h2>
+        <hr />
+        <p>
+          The Logo of the ACPN Conference 2023 will be decided through the
+          annual ACPN logo competition on June 20, 2023.
+        </p>
+        <h4 class="acpn">
+          ACPN NATIONAL <br />
+          CONFERENCE 2023.
+        </h4>
+      </div>
+      <hr class="dash" />
+      <div class="holder">
+        <h2>See the past ACPN Conference</h2>
+        <hr />
+        <p>
+          Take a look at the last two ACPN Conference which took place in Abuja,
+          Nigeria.
+        </p>
+
+        <div id="meeting">
+          <div class="meeting-start">
+            <h3 class="meeting-header">2021</h3>
+            <p class="meeting">ACPN Conference 2021 in Abuja</p>
+          </div>
+
+          <div class="meeting-middle">
+            <h3 class="meeting-header">2019</h3>
+            <p class="meeting">ACPN Conference 2021 in Enugu.</p>
+          </div>
+        </div>
+      </div>
+    </section>
+    <!-- SUB-SECTION ENDS -->
+
+    <!-- PARTNER STARTS -->
+    <section class="partner">
+      <div class="container">
+        <h4>Partners</h4>
+        <hr />
+        <div class="company-logo home-logo">
+          <p class="sponsor1">ASCON</p>
+          <img src="images/Alpha-Logo.jpeg" alt="" />
+          <img src="images/May-Baker-Logo.png" alt="" />
+          <img src="images/Oakleaf-Logo.jpeg" alt="" />
+          <img src="images/sanofi.jpg" alt="" />
+          <img src="images/Bydow-Logo.png" alt="" />
+        </div>
+      </div>
+    </section>
+    <!-- PARTNER ENDS -->
+
+    <!-- FOOTER STARTS -->
+
+    <footer id="footer-about">
+      <div class="footer-content container">
+        <h5>Association of Community Pharmacists Conference, ACPN</h5>
+        <p class="footer-text">
+          2023 Association of Community Pharmacists Conference <br />
+          All Rights Reserved.
+        </p>
+      </div>
+    </footer>
```

Comment:
- Good job done so far 👏, according to the project requirements `All page elements have similar spacing, borders, shadows, and paddings as in` the original design. The footer section of your page has no left and right border. Please fix this 😄. Remember it is important that you try to replicate the Figma design as closely as possible to give confidence to recruiters in your ability to produce exact business requirements 🙏

Your project:
- Please fix this for both mobile and desktop versions 🙏 
![eherhe](https://user-images.githubusercontent.com/73607512/234026443-7dcf084f-a8a0-41cf-8e83-fa90a23484de.PNG)
Design template:
![wrgwerg](https://user-images.githubusercontent.com/73607512/234026680-4848411d-004d-4983-9aa3-dea596339d5f.PNG)